docs/user/index.rst

```diff
@@ -145,7 +144,7 @@ Open source and user focused
     |org_brand| hosts documentation for over 100,000 large
     and small open source projects.
     |com_brand| supports hundreds of organizations with product and internal documentation.
-    Learn more about :doc:`our two platforms </choosing-a-site>`.
+    Learn more about these differences in `our pricing page <https://about.readthedocs.com/pricing/>`_.
```

Comment:
```suggestion
    Learn more about these differences on `our pricing page <https://about.readthedocs.com/pricing/>`_.
```

Also, "our pricing page" feels like an internal name. We could probably avoid this for something more descriptive.

Reply:
"the pricing page" ?

Reply:
```suggestion
    Learn more about these differences on `the pricing page <https://about.readthedocs.com/pricing/>`_.
```

Reply:
To clarify "pricing page" sounds the same to me as "socialaccount view" to me -- we know what it means, but users might not.

"Learn more about these differences on our plan comparison overview" is more descriptive.